python/test/test_api.py

```diff
@@ -135,36 +129,10 @@ def test_memory_fallback_to_cpu(self, server_options):
             fp16_output = numpy.from_dlpack(response.outputs["fp16_output"])
             assert fp16_input[0][0] == fp16_output[0][0]
 
-        tritonserver.default_memory_allocators[tritonserver.MemoryType.GPU] = allocator
-
-    def test_memory_allocator_exception(self, server_options):
-        server = tritonserver.Server(server_options).start(wait_until_ready=True)
-
-        assert server.ready()
-
-        server.load(
-            "test",
-            {
-                "config": json.dumps(
-                    {
-                        "backend": "python",
-                        "parameters": {"decoupled": {"string_value": "False"}},
-                    }
-                )
-            },
-        )
-
-        with pytest.raises(tritonserver.InternalError):
-            for response in server.model("test").infer(
-                inputs={
-                    "string_input": tritonserver.Tensor.from_string_array([["hello"]])
-                },
-                output_memory_type="gpu",
-                output_memory_allocator=TestAllocators.MockMemoryAllocator(),
-            ):
-                pass
-
     def test_unsupported_memory_type(self, server_options):
+        # TODO: Revisit this test when GPU memory support is added, i.e. the request
```

Comment:
Do we have a ticket for GPU follow-up? Can add `DLIS-XXXX` here if so

Reply:
Yes, added the number to the codes: [[comment] Add ticket number to GPU follow-up TODOs](https://github.com/triton-inference-server/core/pull/426/commits/5a307b8d9b43e60773878255a544855df804d129)